improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -847,82 +904,19 @@ def test_weather_data(self):
         data_precip = np.maximum.reduce([data_snow, data_sleet, data_rain])
         data_precipv = np.array(
             [
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                1.0,
-                0.0,
-                0.0,
-                0.0,
-                0.0,
-                1.0,
+                [[1.0, 1.0, 1.0], [1.0, 1.0, 1.0], [1.0, 1.0, 1.0]],
```

Comment:
Could you also do this with the big 1D array at the top of the diff for this file?

Reply:
I've gone through that file and changed ALL the big arrays to be properly structured so that Black formats them nicely.

Reply:
Thank you :)